src/genie/libs/parser/nxos/tests/test_show_fabricpath.py

```diff
@@ -0,0 +1,60 @@
+# Import the Python mock functionality
+import unittest
+from unittest.mock import Mock
+
+# pyATS
+from pyats.topology import Device
+from pyats.topology import loader
+
+# Metaparser
+from genie.metaparser.util.exceptions import SchemaEmptyParserError, SchemaMissingKeyError
+
+# nxos show_fabricpath
+from genie.libs.parser.nxos.show_fabricpath import ShowFabricpathIsisAdjacency
+
+# =================================
+# Unit test for 'show fabricpath isis adjacency'
+# =================================
+class test_show_fabricpath_isis_adjacency(unittest.TestCase):
```

Comment:
This should be TestShowFabricpathIsisAdjacency.

Reply:
Based on your example : 

https://pubhub.devnetcloud.com/media/pyats-development-guide/docs/writeparser/writeparser.html#test-your-new-parser

Reply:
Our team is now update all the class names in unittests to follow PEP8. We will update the name later. and we will update the doc as well.